core/shared/src/main/scala/cats/effect/concurrent/MVar.scala

```diff
@@ -52,6 +54,7 @@ import cats.~>
  * Inspired by `Control.Concurrent.MVar` from Haskell and
  * by `scalaz.concurrent.MVar`.
  */
+@deprecated("`MVar` is now deprecated in favour of a new generation `MVar2` with `tryRead` support", "2.1.0")
```

Comment:
The version has progressed since we started talking about this:

```suggestion
@deprecated("`MVar` is now deprecated in favour of a new generation `MVar2` with `tryRead` support", "2.2.0")
```

Reply:
Awesome! Updated